sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/link.py

```diff
@@ -97,6 +97,8 @@ def __init__(
         self._on_link_state_change = kwargs.get("on_link_state_change")
         self._on_attach = kwargs.get("on_attach")
         self._error: Optional[AMQPLinkError] = None
+        self._drain_state = False
+        self._received_drain_response = False
```

Comment:
I don't see where this is used?

Reply:
it is used in outgoing/incoming disp and in receive_loop in pyamqp_transport